www/js/app.js

```diff
@@ -64,6 +64,10 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
      */
     var selectedArchive = null;
     
+    if(!window.matchMedia) document.getElementById('appThemeSelect').options[0].style.display = "none";
+    if(!window.matchMedia) document.getElementById('appThemeSelect').options[1].style.display = "none";
+    var darkPreference = window.matchMedia('(prefers-color-scheme:dark)');
```

Comment:
Move these handlers into uiUtil.applyAppTheme as well. You will be calling that function in a moment, so you can set up the handler for the matchMedia change there, and it will be a lot neater.